R/analytics-app.R

```diff
@@ -71,6 +70,7 @@ analytics_ui <- function() {
   )
 
   final_stats <- shiny.semantic::segment(
+    class = "centered-and-margined full-width",
```

Comment:
Minimal way of making sure the segments have breathing room and occupy the full width. I was not sure why some segments need to be side-by-side since they still need more horizontal space once the plots are rendered.